src/pyFAI/detectors/_common.py

```diff
@@ -387,10 +387,12 @@ def get_config(self):
             dico["sensor"] = self.sensor.as_dict()
         return dico
 
-    def get_splineFile(self):
+    @property
+    def splineFile(self) -> str:
```

Comment:
What is the coding convention, CamelCase or PEP8?

```suggestion
    def spline_file(self) -> str:
```